### PR TITLE
adding info log for data sizes

### PR DESCRIPTION
**Pull request by stanislav-bedunkevich**

### CODE REVIEW

**Review by AndrewNepogoda — APPROVED** [review]

File: workflow/executor.py [hunk 1]
```diff
@@ -185,6 +186,8 @@ def _inflate_sources(self, logger: logging.Logger, rai_config: RaiConfig, env_co
             inflated_paths = self.paths_builders[src.container.name].build(logger, days, src.relative_path,
                                                                            src.extensions,
                                                                            src.is_date_partitioned)
+            if src.is_support_data_size_print():
```
Comment (stanislav-bedunkevich) [inline workflow/executor.py:189]:
```suggestion
            if src.has_size_attribute():
```
maybe this way better?

Comment (AndrewNepogoda) [reply workflow/executor.py:189]: size is not a source attribute

Comment (AndrewNepogoda) [reply workflow/executor.py:189]: @stanislav-bedunkevich maybe `is_size_supported()`

**Review by stanislav-bedunkevich — COMMENTED** [review]
Thanks!

**Pull request merged**